<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BuysPage</class>
 <widget class="QWidget" name="BuysPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>664</width>
    <height>342</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SlingRoad Buys</string>
  </property>
  <layout class="QVBoxLayout" name="topLayout">
   <property name="leftMargin">
    <number>10</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>10</number>
   </property>
   <property name="bottomMargin">
    <number>10</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>My Buys:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableWidget" name="tableWidget">
       <property name="editTriggers">
        <set>QAbstractItemView::NoEditTriggers</set>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Status</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Vendor</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Item</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Item Id</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Request Id</string>
        </property>
       </column>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QPushButton" name="deliveryDetailsButton">
       <property name="text">
        <string>Send &amp;Delivery Details</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="escrowLockButton">
       <property name="text">
        <string>&amp;Escrow Lock</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="payButton">
       <property name="text">
        <string>&amp;Pay / Release Escrow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="refundButton">
       <property name="text">
        <string>&amp;Request Refund</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="copyAddressButton">
       <property name="text">
        <string>&amp;Copy Seller Address</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
